plan/typeinferer.go

```diff
@@ -274,7 +274,7 @@ func (v *typeInferrer) handleFuncCallExpr(x *ast.FuncCallExpr) {
 		}
 	case "interval":
 		tp = types.NewFieldType(mysql.TypeLonglong)
-	case "ceil", "ceiling":
+	case "ceil", "ceiling", "floor":
```

Comment:
I tried. But I could not find ceil in this test file. This is kinda confused me. If I add test for floor, then so does ceil and ceiling.